CHANGELOG.txt

```diff
@@ -1,7 +1,7 @@
 v4.0.2
--ph5.clients.ph5toms
+-ph5.clients.ph5toms (issue #100)
  * Fixed memory leak bug and reduced memory usage
--ph5.clients.ph5tostationxml
+-ph5.clients.ph5tostationxml (issue #107)
  * added receiver-id to obspy inventory and added an extra atribute at teh station level
```

Comment:
@derick-hess Oops I notices a few small typos "teh" instead of "the" and "atribute" instead of "attribute"